.circleci/config.yml

```diff
@@ -99,7 +99,7 @@ commands:
       - run:
           name: Run maven package
           command: |
-            mvn package -DskipTests -file << parameters.pom_file >> -Dcheckstyle.skip -Dmaven.repo.local=<< parameters.m2_repo >>
+            mvn package -Dmaven.test.skip -file << parameters.pom_file >> -Dcheckstyle.skip -Dmaven.repo.local=<< parameters.m2_repo >>
```

Comment:
`-Dmaven.test.skip` will skip both compiling and running tests. I wanted to apply this to the rest of this file, but some commands actually needed to compile the test classes, so I didn't.